pkgs/tools/filesystems/irods/common.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv, bzip2, zlib, autoconf, automake, cmake, gnumake, help2man , texinfo, libtool , cppzmq , libarchive, avro-cpp, boost, jansson, zeromq, openssl, pam, libiodbc, kerberos, gcc, libcxx, cross, which }:
+
+# Common attributes of irods packages
+
+with stdenv;
+
+# Boost and avro-c++ have to be built with the same stdenv
```

Comment:
@FRidh :
Boost depends on avro-cpp.
Irods depends on boost.
I have to build irods with llvm38 and libcxx, see the call:
```
inherit (callPackages ../tools/filesystems/irods {
          stdenv = llvmPackages_38.libcxxStdenv;  
          libcxx = llvmPackages_38.libcxx;        
          boost = boost160;                       
        })                                        
    irods                                         
    irods-icommands;                              
```
But for it to work, boost needs also to be built with llvm38 and libcxx, and avro-cpp too.
If you see a better way to do this, I would be very glad if you could help me doing this way...

I really need this package to be merged...

Thank you